License2Deploy/rolling_deploy.py

```diff
@@ -52,6 +52,7 @@ def __init__(self,
     self.health_wait = health_wait
     self.only_new_wait = only_new_wait
     self.existing_instance_ids = []
+    self.old_desired_capacity = 2
     self.new_desired_capacity = None
```

Comment:
since it's only used in the one function, calculate_autoscale_desired_instance_count, it doesn't need to be declared in the init. we can just declare it in the function w/o `self.`